lib/rubocop/cop/style/word_array.rb

```diff
@@ -82,10 +82,16 @@ class << self
           attr_accessor :largest_brackets
         end
 
+        def on_new_investigation
+          super
+
+          reset_matrix_of_complex_content_cache!
+        end
```

Comment:
Up to you, but `super` is not needed in this case and I would inline `reset_matrix_of_complex_content_cache!` here.

Reply:
https://github.com/rubocop/rubocop/blob/master/lib/rubocop/cop/base.rb#L135-L139 specifically says to always call `super`, which is why I did it, and extracted the method.

Fine with inlining though, if you think that's better. It just felt weird to have logic in `on_new_investigation`.

Reply:
> specifically says to always call super, which is why I did it, and extracted the method.

Ah, good, looks like this is currently missing in most of the places.

> It just felt weird to have logic in on_new_investigation

Seems like thats how it is done throughout the other places.